index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html style=" background-color:aqua;
+text-align:center;
+" >
+    <head>
+    
+    <link rel="stylesheet" href="project-1.css">
```

Comment:
the separate css file is connected but no line of codes is used in the file. use the file to apply styles in  web page for more appealing.